### PR TITLE
API sketch for X509 cert chain verification

### PR DESCRIPTION
**Pull request by kislyuk**
See #77 for discussion

### CODE REVIEW

File: src/truststore/hazmat/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""
```
Comment (sethmlarson) [inline src/truststore/hazmat/__init__.py:1]: Let's lift this prose into it's own documentation page as a warning?

**Review by sethmlarson — COMMENTED** [review]
In general I'm happy with this design, I'd like to lose the creation of an `ssl.SSLContext` object if possible. I also wonder how we're going to configure certificate verification for platforms, things like CRLs, etc?

**Comment by sethmlarson** [comment]
Hey @kislyuk, I have some more time to spend on OSS now. Are you interested in picking this PR back up?

**Comment by kislyuk** [comment]
I am definitely interested in getting this PR finished but would welcome any help! I had to pause work on it because of too many other things going on. If you want to take over any part of it I would very much appreciate it, otherwise I will try to get back to it over the course of the next few weekends.

**Comment by sethmlarson** [comment]
Hey @kislyuk, since it's been a bit on this PR I am not sure if you'll get to this one (no worries if not!) I'm going to close it but let me know if you'd like to pick it back up.

**Pull request closed without merging**